regression/cbmc/replace-calls-01/main.c

```diff
@@ -0,0 +1,19 @@
+#include <assert.h>
```

Comment:
Random place: the description provided in the PR is way more useful than the commit message - could the latter please be updated?